code-challenges.js

```diff
@@ -31,6 +31,31 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const boilingPoint = (number) =>{
+    /* The purpose of this funtion is to input a number and return whether that number is at, below, or above our boiling temp of 212. Then return said number and its boiling state. Eg:
+
+    const temperature1 = 42
+    Expected output: "42 is below boiling point"
+
+    const temperature2 = 350
+    Expected output: "350 is above boiling point"
+
+    const temperature3 = 212 
+    Expected output: "212 is at boiling point"
+    */
+    let res = ''
+    if (number > 212){
+        res = `${number} is above boiling point`
+    }
+    else if (number === 212){
+        res = "212 is at boiling point"
+    }
+    else {
+        res = `${number} is below boinling point`
+    }
+    return res 
+}
```

Comment:
Nice! No need to make the `res` variable at all, you can just throw return in for each string that each condition is outputting.